backend/src/main/java/com/woowacourse/moragora/dto/ParticipantResponse.java

```diff
@@ -0,0 +1,42 @@
+package com.woowacourse.moragora.dto;
+
+import com.woowacourse.moragora.entity.Status;
+import com.woowacourse.moragora.entity.user.User;
+import java.util.Locale;
+import lombok.Getter;
+
+@Getter
+public class ParticipantResponse {
+
+    private final Long id;
+    private final String email;
+    private final String nickname;
+    private final String attendanceStatus;
+    private final int tardyCount;
+
+    private ParticipantResponse(final Long id,
+                                final String email,
+                                final String nickname,
+                                final String attendanceStatus,
+                                final int tardyCount) {
+        this.id = id;
+        this.email = email;
+        this.nickname = nickname;
+        this.attendanceStatus = attendanceStatus;
+        this.tardyCount = tardyCount;
+    }
+
+    public ParticipantResponse(final Long id,
+                               final String email,
+                               final String nickname,
+                               final Status attendanceStatus,
+                               final int tardyCount) {
+        this(id, email, nickname,
+                attendanceStatus.name().toLowerCase(Locale.ROOT), tardyCount);
+    }
+
+    public static ParticipantResponse of(final User foundUser, final Status attendanceStatus, final int tardyCount) {
+        return new ParticipantResponse(foundUser.getId(), foundUser.getEmail(), foundUser.getNickname(),
+                attendanceStatus.name().toLowerCase(Locale.ROOT), tardyCount);
```

Comment:
```suggestion
                return new ParticipantResponse(foundUser.getId(), foundUser.getEmail(), foundUser.getNickname(),
                attendanceStatus, tardyCount);
```
로 해서 `name().toLowerCase(Locale.ROOT)` 를 줄이는 게 어떨까요?

Reply:
static factory method에서는 Status 객체를 param type으로 가지는 부생성자를 사용하게 하고, 소문자 변환은 주생성자에서 마지막으로 처리하게 해서 중복 로직을 제거했어요.
소문자 변환을 주생성자에게 처리하게 한 이유는, 대소문자 여부가 API 동작에 큰 영향을 미치기 때문에 반드시 처리됨을 보장하기 위해서 가장 마지막에 호출되는 주생성자에게 주었습니다.